recipe/meta.yaml

```diff
@@ -70,10 +68,12 @@ requirements:
     - {{ mpi }}  # [mpi != 'nompi']
     - libnetcdf  # [tempest == 'tempest']
     - libnetcdf * {{ mpi_prefix }}_*  # [tempest == 'tempest']
-    - netcdf-cxx-legacy  # [tempest == 'tempest']
-    - netcdf-cxx-legacy * {{ mpi_prefix }}_*  # [tempest == 'tempest']
     - eigen
-    - tempest-remap 2.0.2  # [tempest == 'tempest']
+    - tempest-remap 2.0.5  # [tempest == 'tempest']
+    - metis
+    - libblas  # [tempest == 'tempest']
+    - liblapack  # [tempest == 'tempest']
```

Comment:
I added `libblas` and `liblapack` only for tempest builds.  I'm not sure that's what we want but it makes the most sense to me.

I don't think we want to explicitly ask for `openblas` because `tempest-remap` was only built with the `netlib` version so I don't think that will turn out well.  If we do explicitly need `openblas`, we should figure out how to get `tempest-remap` building with both.  That being said, I don't know too much about how BLAS works so maybe mixing and matching is fine somehow?